### DISCUSSION TITLE
Assets download path invalid with subfolder

### DISCUSSION THREAD

**Discussion by alrick** [directus/directus#15122] [Archive]
### Describe the Bug

When directus is configured with a subfolder, the dowload path from assets within collection are wrong, it's missing the subfolder.
It's working correctly when dowloading the asset from the file library.

### To Reproduce

* Configure a self hosted directus with a subfolder config : example.com/directus
* Create a collection with an asset (eg : image)
* Create an item within collection with asset
* On item page, try to download the asset

### Errors Shown

Address of the entry :
![image](https://user-images.githubusercontent.com/1086355/185100104-1e049d3e-7975-496f-ad8c-a0ddb03f3949.png)

Error logged to console when trying to access image :
![image](https://user-images.githubusercontent.com/1086355/185100221-2269e8a8-9740-4ab2-aa12-d5a010a21eb4.png)

### What version of Directus are you using?

9.15.1

### What version of Node.js are you using?

16.16.0

### What database are you using?

SQLite 3.37.2

### What browser are you using?

Mozilla Firefox, Brave

### How are you deploying Directus?

CLI installation

**Comment by rijkvanzanten** [comment]
What's the value for the `PUBLIC_URL` environment variable?

**Reply by alrick** [reply]
`https://eddb9.unifr.ch/diatheque` I tried `/diatheque` too.
But if I try to download the asset from the files manager, it's working as excepted.

**Reply by rijkvanzanten** [reply]
When you say "On item page, try to download the asset", do you mean by clicking the download button in the image interface, or?

**Reply by alrick** [reply]
![Screenshot from 2022-08-17 17-53-31](https://user-images.githubusercontent.com/1086355/185186265-8d9e3c36-567a-4382-95a4-a5ddde63a748.png)
I hope it's more clear that way.

**Reply by rijkvanzanten** [reply]
Yeah! Thanks for the info 🙂

**Comment by licitdev** [comment]
This should be fixed in https://github.com/directus/directus/pull/15126 :)
